### PR TITLE
Release 1.2 News file

### PR DESCRIPTION
**Pull request by HenryMiller1**
I've been meaning to do this for a while...   Here are some news.  I'll leave this open for a day (I hope) just to see if anyone wants to review this.  I think I put the interesting things in here, but if anyone has something they feel strongly about that I didn't put in now is your chance.

### CODE REVIEW

**Comment by HenryMiller1** [comment]
When I actually merge this (tomorrow?) I'll plan also updating the version number and doing a release

File: NEWS [hunk 1]
```diff
@@ -1,5 +1,16 @@
 1.2 (in progress)
-
+ - Add more compiler flags to the list that mean build locally
+ - Make load calculation better
+ - Limit amount of data sent at one time for slow networks/remotes
+ - Many updates to the tests
+ - Better logs of some error condidtions
+ - Better decide if build should be local or remote
```
Comment (ossilator) [inline NEWS]: that's some dodgy grammar.

File: NEWS [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+ - Support adding gcc and clang to the same enviornment
```
Comment (ossilator) [inline NEWS]: typo

File: NEWS [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+ - Better handling of icerun
+ - Cygwin now works as a client
+ - Don't expose Host endian to network
```
Comment (ossilator) [inline NEWS]: endianness

File: NEWS [hunk 4]
```diff
@@ -4,11 +4,11 @@
  - Limit amount of data sent at one time for slow networks/remotes
  - Many updates to the tests
  - Better logs of some error condidtions
```
Comment (ossilator) [inline NEWS]: oh, another typo

File: NEWS [hunk 5]
```diff
@@ -1,5 +1,16 @@
 1.2 (in progress)
-
+ - Add more compiler flags to the list that mean build locally
```
Comment (gang65) [inline NEWS]: It would be great to list additional flags which force to build locally.

**Pull request merged**